src/neural/cache.h

```diff
@@ -73,6 +73,8 @@ class CachingComputation {
   float GetQVal(int sample) const;
   // Returns P value @move_id of @sample.
   float GetPVal(int sample, int move_id) const;
+  // Pops last input from the computation. Only allowed for inputs from cache.
```

Comment:
When I first reviewed, "inputs from cache" slightly confused me. I understand better now having read the whole thing, but probably still better to tweak this comment to say "Only allowed for inputs which were cached" or "inputs which were cache hits" or something like that, the "inputs *from* cache" just confused me ever so slightly.

Reply:
Done.